protocol/message.go

```diff
@@ -216,6 +262,40 @@ func NewMonitoringProof(ap []*m.AuthenticationPath,
 	}, ReqSuccess
 }
 
+// NewObservedSTR creates the response message a CONIKS auditor
+// sends to a client, or a CONIKS directory sends to an auditor,
```

Comment:
If we use a pub-sub architecture, will this be used between an auditor and a directory?